torch/onnx/utils.py

```diff
@@ -1015,6 +1015,15 @@ def _should_aten_fallback(ns, op_name, opset_version, operator_export_type):
     return is_onnx_aten_export or (not is_exportable_aten_op and is_aten_fallback_export)
 
 
+def _need_symbolic_context(symbolic_fn):
+    # Check if the first argument to symbolic_fn is annotated as type `torch.onnx.SymbolicContext`
+    full_arg_spec = inspect.getfullargspec(symbolic_fn)
+    args = full_arg_spec.args
+    annotations = full_arg_spec.annotations
+    if len(args) == 0:
+        return False
+    return args[0] in annotations and annotations[args[0]] == SymbolicContext
```

Comment:
Probably not very important, but I suggest allowing subclasses:
`and issubclass(annotations[args[0]], SymbolicContext)`